src/components/body.js

```diff
@@ -51,7 +51,7 @@ class TimelineBody extends Component {
 TimelineBody.propTypes = {
   width: PropTypes.number.isRequired,
   columnWidth: PropTypes.func.isRequired,
-  height: PropTypes.number.isRequired,
+  height: PropTypes.oneOfType([PropTypes.number, PropTypes.func]).isRequired,
```

Comment:
Is this required? Looks like we still pass in a number

Reply:
We pass the function calculateHeight. I can remove number as required.

Reply:
Not sure If i'm missing something here...
The props is:
```javascript
height={calculateHeight(height)}
```
and `calculateHeight` returns a number, so the prop type is a number?

Reply:
You are right, I will remove ProtTypes.func as required.